### DISCUSSION TITLE
使用案例指令评估LLaMA-7b模型在MMLU和C-Eval数据集上的性能时评测结果为空

### DISCUSSION THREAD

**Discussion by liujunfei678** [open-compass/opencompass#590] [Q&A]
案例：
按照上述步骤确保OpenCompass正确安装并准备好数据集后，您可以使用以下命令评估LLaMA-7b模型在MMLU和C-Eval数据集上的性能：
`python run.py --models hf_llama_7b --datasets mmlu_ppl ceval_ppl`

结果：
![image](https://github.com/open-compass/opencompass/assets/134382914/7e39d88c-1d57-4937-84ed-ec7be4a31c5c)

过程中存在的报错：

`11/14 15:44:18 - OpenCompass - ERROR - /home/liujunfei/opencompass/opencompass/runners/base.py - summarize - 63 - OpenICLInfer[llama-7b-hf/lukaemon_mmlu_professional_law_0] failed with code 1`
`11/14 15:44:18 - OpenCompass - ERROR - /home/liujunfei/opencompass/opencompass/runners/base.py - summarize - 63 - OpenICLInfer[llama-7b-hf/lukaemon_mmlu_professional_law_1] failed with code 1`

我的问题：
我跟着opencompass的安装指南一步步操作下来，不知道为什么出现了** 评测结果为空**，且出现了一些我不理解的**报错消息**。

**Comment by tonysy** [comment]
Please check the log in *.out file

**Reply by liujunfei678** [reply]
我看了一下四个out日志文件
大概是由于在建立与“huggingface.co”的安全时发生了SSL错误。“SSL: UNEXPECTED_EOF_WHILE_READING”
但我不知道这个要怎么解决，是我的防火墙的问题吗？

**Reply by tonysy** [reply]
It seems the erro is caused by the Intenet connection of Huggingface. If you have downloaded the checkpoints before, please use the specific path of the checkpoint, and use ` HF_EVALUATE_OFFLINE=1 HF_DATASETS_OFFLINE=1 TRANSFORMERS_OFFLINE=1` to avoid the network error

**Comment by YangYu-Li** [comment]
您好，您遇到的这个问题解决了吗？

**Comment by christyzou** [comment]
我也遇到了相同的问题
